release-notes.txt

```diff
@@ -1,3 +1,365 @@
+RIOT-2019.04 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2019.04 release includes a number of new features including improved
+support for ESP boards, porting of riotboot to a number of new platforms,
```

Comment:
There are no new board definitions for ESP boards in this release.

Reply:
The idea was to reflect improved functionality on existing boards, rather than implying that we have added new boards

Reply:
Would you like me to clarify the statement or remove?

Reply:
> The idea was to reflect improved functionality on existing boards, rather than implying that we have added new boards

Hm ... IMO, noticeable improvements are only esp-now network device driver and freed RAM for ESP8266 which are listed under the `Boards` section. I would prefer to remove it.

```diff
- support for ESP boards, porting of riotboot to a number of new platforms,
+ porting of riotboot to a number of new platforms,
```

Reply:
Changed and pushed